cmake/oss.cmake.in

```diff
@@ -178,8 +169,8 @@ ExternalProject_Add(fmt
      -DFMT_TEST=OFF)
 
 ExternalProject_Add(seastar
-  GIT_REPOSITORY https://github.com/redpanda-data/seastar.git
-  GIT_TAG bdf4c58630f06aaffa02c98edc1192a2d8db2fe8
+  GIT_REPOSITORY https://github.com/BenPope/seastar.git
```

Comment:
This should probably be the same as in the vtools pr (redpanda-data and a sha1), right?